ioctl/cmd/account/accountactions.go

```diff
@@ -0,0 +1,202 @@
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package account
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/golang/protobuf/ptypes"
+	"github.com/grpc-ecosystem/go-grpc-middleware/util/metautils"
+	"github.com/rodaine/table"
+	"github.com/spf13/cobra"
+	"google.golang.org/grpc/status"
+
+	"github.com/iotexproject/iotex-core/ioctl/config"
+	"github.com/iotexproject/iotex-core/ioctl/output"
+	"github.com/iotexproject/iotex-core/ioctl/util"
+	"github.com/iotexproject/iotex-proto/golang/iotexapi"
+	"github.com/iotexproject/iotex-proto/golang/iotextypes"
+)
+
+var countLimit *uint64
+
+// Multi-language support
+var (
+	actionsCmdShorts = map[config.Language]string{
+		config.English: "Show the list of actions for an account",
+		config.Chinese: "显示账户的操作列表",
+	}
+	actionsCmdUses = map[config.Language]string{
+		config.English: "actions (ALIAS|ADDRESS)  [SKIP]",
+		config.Chinese: "actions (ALIAS|ADDRESS)  [SKIP]",
+	}
+	flagCountUsages = map[config.Language]string{
+		config.English: "choose a count limit",
+		config.Chinese: "选择一个计数限制",
+	}
+)
+
+// accountActionsCmd represents the account sign command
+var accountActionsCmd = &cobra.Command{
+	Use:   config.TranslateInLang(actionsCmdUses, config.UILanguage),
+	Short: config.TranslateInLang(actionsCmdShorts, config.UILanguage),
+	Args:  cobra.RangeArgs(1, 2),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		cmd.SilenceUsage = true
+		err := accountActions(args)
+		return output.PrintError(err)
+	},
+}
+
+func init() {
+	countLimit = accountActionsCmd.Flags().Uint64("count", 15, config.TranslateInLang(flagCountUsages, config.UILanguage))
+}
+
+func accountActions(args []string) error {
+	var skip uint64
+	var err error
+	if len(args) == 2 {
+		skip, err = strconv.ParseUint(args[1], 10, 64)
+		if err != nil {
+			return output.NewError(output.ConvertError, "failed to convert skip ", nil)
+		}
+	}
+
+	addr, err := util.Address(args[0])
+	if err != nil {
+		return output.NewError(output.AddressError, "failed to get address", err)
+	}
+
+	conn, err := util.ConnectToEndpoint(config.ReadConfig.SecureConnect && !config.Insecure)
+	if err != nil {
+		return output.NewError(output.NetworkError, "failed to connect to endpoint", err)
+	}
+	defer conn.Close()
+	cli := iotexapi.NewAPIServiceClient(conn)
+	ctx := context.Background()
+
+	jwtMD, err := util.JwtAuth()
+	if err == nil {
+		ctx = metautils.NiceMD(jwtMD).ToOutgoing(ctx)
+	}
+
+	requestGetAccount := iotexapi.GetAccountRequest{
+		Address: addr,
+	}
+	accountResponse, err := cli.GetAccount(ctx, &requestGetAccount)
+	numActions := accountResponse.AccountMeta.GetNumActions()
+	fmt.Println("Total:", numActions)
```

Comment:
The 'Total' may confuse users if they don't realize the default setting for '--count' flag. They may consider it as number of returned actions. The first I saw your example, I thought there would be 60 TXs printed.

Reply:
Maybe we can use '--limit' instead

Reply:
Normal users may not read the flag usage. Printing a 'returned' or 'printed' number the same time may be a better solution.

Reply:
thx @Frankonly help review

1. i think we can just remove this? if user input 35 (like your example), then print out 35 actions
2. what if user did not input 35, will it print out all actions?

Reply:
@dustinxie in my example, 35 means skip latest 35 records, equal to getActions(start=60-35-15 ,count=15)

see command help 
```
ioctl account actions -h
Show the list of actions for an account

Usage:
  ioctl account actions (ALIAS|ADDRESS)  [SKIP] [flags]

Flags:
      --count uint   choose a count limit (default 15)
  -h, --help         help for actions

Global Flags:
      --endpoint string        set endpoint for once (default "api.testnet.iotex.one:443")
      --insecure               insecure connection for once
  -o, --output-format string   output format
```

Reply:
`ioctl account actions io1fuhhg9jgdxwpms9dsdfwjdc90nt7v67hx40cd8` = https://testnet.iotexscan.io/address/io1fuhhg9jgdxwpms9dsdfwjdc90nt7v67hx40cd8

`ioctl account actions io1fuhhg9jgdxwpms9dsdfwjdc90nt7v67hx40cd8 45` = click page 4

Reply:
change `--count` to `--limit` may be proper for pagination semantics